bin/configs/dart-dio-echo-server-dio-built_value.yaml

```diff
@@ -0,0 +1,18 @@
+generatorName: dart-dio
+outputDir: samples/client/echo_api/dart/dart-dio-built_value
+inputSpec: modules/openapi-generator/src/test/resources/3_0/dart/echo_api.yaml
+templateDir: modules/openapi-generator/src/main/resources/dart/libraries/dio
+typeMappings:
+  Client: "ModelClient"
+  File: "ModelFile"
+  EnumClass: "ModelEnumClass"
+additionalProperties:
+  hideGenerationTimestamp: "true"
+  enumUnknownDefaultCase: "true"
+  serializationLibrary: "built_value"
+  networkingLibrary: "dio"
```

Comment:
What about using `library` (global option, not generator option) instead?

Reply:
currently the `serializationLibrary` is assigned to `library` global option (this was introduced in https://github.com/OpenAPITools/openapi-generator/pull/9980)
I was going to make another PR that targets v7.0.0, with the correct parameters assigned, but I am not sure if this is the right approach

Reply:
relevant line: https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/DartDioClientCodegen.java#L162